YumemiTraining/Model/WeatherUseCase.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  YumemiWetherAPIModel.swift
+//  YumemiTraining
+//
+//  Created by 小西 俊輔 on 2021/03/29.
+//
+
+import Foundation
+import YumemiWeather
+
+
+final class WeatherUseCase {
+    static let weatherDidReloadNotification = Notification.Name("weatherReload")
+    
+    func reloadWeather() {
+        guard let weather =  WeatherType(rawValue: YumemiWeather.fetchWeather()) else {
+            return
+        }
+        NotificationCenter.default.post(name: WeatherUseCase.weatherDidReloadNotification, object: nil, userInfo: ["weather": weather])
```

Comment:
[nits]
自分自身の型を使うときは `Self` 使えるので便利ですー

```suggestion
        NotificationCenter.default.post(name: Self.weatherDidReloadNotification, object: nil, userInfo: ["weather": weather])
```